backend/CHANGELOG.rst

```diff
@@ -7,6 +7,10 @@ This file keeps track of all notable changes to license-manager-backend
 Unreleased
 ----------
 
+2.2.20 -- 2023-02-07
+--------------------
+* Fix overbooking issue
```

Comment:
```suggestion
* Fix overbooking issue by ....
```

Reply:
Thank you, I updated the CHANGELOG message.